brushfire-tree/src/main/scala/com/stripe/brushfire/Error.scala

```diff
@@ -1,7 +1,20 @@
 package com.stripe.brushfire
 
-import com.twitter.algebird._
-
+import com.twitter.algebird.{AveragedValue, Group, Monoid, Semigroup}
+
+/** Computes some kind of error by comparing the trees' predictions to the validation set */
+trait Error[T, P, E] {
```

Comment:
Though TBH I can see an argument for including pruning and error here, also - basically, even though they are part of the training phase, they are likely to be things we want, both in interface and implementation, for *any* training approach, vs all of the splitter/evaluator/etc machinery which are very specific to the particular approach we're taking. So, fine, leave it :)